### PR TITLE
7899 currency not showing

### PR DESCRIPTION
**Pull request by arisah-ijeoma**

### CODE REVIEW

**Comment by emitche** [comment]
Polish:
Remove the extra space from the transactions.

Testing this inside the browser, there is a space between the currency and the amount. For US $, I am expecting `US $0.00`.

For other currencies often used in the system, Argentina is displayed as `AR $0.00` in the loan details. Nicaragua is displayed as `NI $0.00` at the moment in loan details. So we should do something similar in the transactions table.

This is what is currently in transactions:
image

**Comment by emitche** [comment]
I think #459 fixes the above space issue.

**Comment by emitche** [comment]
Merging develop into this branch will fix the spacing concern noted above.

File: app/models/accounting/quickbooks/updater.rb [hunk 1]
```diff
@@ -111,6 +113,16 @@ def extract_qb_data(txn)
         txn.save!
       end
 
+      def lookup_currency(txn)
+        project = txn.project
+
+        if txn.quickbooks_data && txn.quickbooks_data[:currency_ref]
```
Comment (linesarefuzzy) [inline app/models/accounting/quickbooks/updater.rb]: I think you can do `.try(:[], :currency_ref)` here

File: app/models/accounting/quickbooks/updater.rb [hunk 2]
```diff
@@ -111,6 +113,16 @@ def extract_qb_data(txn)
...
+          Currency.find_by(code: quickbooks_data[:currency_ref][:value]).try(:id)
+        elsif project
+          Currency.find_by(id: project.currency_id)
```
Comment (linesarefuzzy) [inline app/models/accounting/quickbooks/updater.rb]: Can't you just use `project.currency` here?

File: app/models/accounting/quickbooks/updater.rb [hunk 3]
```diff
@@ -111,6 +113,16 @@ def extract_qb_data(txn)
         txn.save!
       end
 
+      def lookup_currency(txn)
+        project = txn.project
```
Comment (linesarefuzzy) [inline app/models/accounting/quickbooks/updater.rb]: This variable seems unnecessary to me

File: app/models/accounting/quickbooks/updater.rb (hunk 2 above)
Comment (arisah-ijeoma) [reply app/models/accounting/quickbooks/updater.rb]: I could :)
thanks

File: app/models/accounting/quickbooks/updater.rb (hunk 3 above)
Comment (arisah-ijeoma) [reply app/models/accounting/quickbooks/updater.rb]: I just felt weird calling `txn.project` in two different places

**Pull request merged**